html/datasets.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Dataset Browser</title>
+
+    <!--<script type="text/javascript" src="facetview2/vendor/jquery/1.7.1/jquery-1.7.1.min.js"></script>-->
+    <!--<link rel="stylesheet" href="facetview2/vendor/bootstrap/css/bootstrap.min.css">-->
+    <!--<script type="text/javascript" src="facetview2/vendor/bootstrap/js/bootstrap.min.js"></script>-->
+    <!--<link rel="stylesheet" href="facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.css">-->
+    <!--<script type="text/javascript" src="facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.min.js"></script>-->
+
+    <!--<script type="text/javascript" src="facetview2/es.js"></script>-->
+    <!--<script type="text/javascript" src="facetview2/bootstrap2.facetview.theme.js"></script>-->
+    <!--<script type="text/javascript" src="facetview2/jquery.facetview2.js"></script>-->
+    <!--<link rel="stylesheet" href="facetview2/css/facetview.css">-->
+    <!--<link rel="stylesheet" href="browsers.css">-->
+
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/jquery/1.7.1/jquery-1.7.1.min.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/vendor/bootstrap/css/bootstrap.min.css">
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/bootstrap/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.css">
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/jquery-ui-1.8.18.custom/jquery-ui-1.8.18.custom.min.js"></script>
+
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/es.js"></script>
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/bootstrap2.facetview.theme.js"></script>
+    <script type="text/javascript" src="/vivo/themes/dco/js/facetview2/jquery.facetview2.js"></script>
+    <link rel="stylesheet" href="/vivo/themes/dco/js/facetview2/css/facetview.css">
+    <link rel="stylesheet" href="/vivo/themes/dco/css/browsers.css">
+
+
+    <script type="text/javascript">
+        jQuery(document).ready(function($) {
+            $('.facet-view-simple').facetview({
+//                search_url: '//localhost:9200/dco/dataset/_search',
+                search_url: '//dcotest.tw.rpi.edu/search/dco/dataset/_search',
+                page_size: 50,
+                sort: [
+                    {"_score" : {"order" : "desc"}}
```

Comment:
Well, the _score depends on the query when we get the data from ES, and serves as, say, "relevance", to the criterions; since we are getting all the data with a plain search, all the _score's in result is 1... 
If we want to use the _score as the parameter to sort, we will need to have a search_url like:
'//dcotest.tw.rpi.edu/search/dco/dataset/_search?q=blablablablabla....'
So you want it to be sorted in, say, publicationYear, in a way that we edit the search_url?